nion/utils/ListModel.py

```diff
@@ -285,23 +289,22 @@ def __init__(self, *, container: typing.Optional[Observable.Observable] = None,
         self.__container = None
         self.__master_items_key = master_items_key or items_key or "items"
         self.__items_key = items_key or "items"
-        self.__master_items: typing.List[typing.Any] = list()  # a list of source items (to be filtered)
+        self.__master_items = list[typing.Any]()  # a list of source items (to be filtered)
```

Comment:
I'm not sure why this change was made as I don't think it affects anything, but should the same be done for self.__items for consistency?